webapp/src/pages/Settings.tsx

```diff
@@ -689,6 +689,9 @@ const Settings: React.FC<Props> = ({ onClose }) => {
       <DialogContent
         dividers
         sx={{
+          [`& .${formControlLabelClasses.root}`]: {
+            width: "fit-content",
+          },
           [`& .${formControlLabelClasses.labelPlacementStart}`]: {
             justifyContent: "flex-end",
```

Comment:
I think it would be best to work with alignment instead of width. It might be fine with `width`, but I'm afraid it might cause unexpected behaviors.
Since we are inside a "flex" container (`FormGroup`) with the default `align-items: stretch`, we could overwrite this for the `formControlLabelClasses.root` with `align-self: start`.
What's cool is that this works for `formControlLabelClasses.labelPlacementStart` too, to which we can remove the `justify-content: flex-end`.
```suggestion
          [`& .${formControlLabelClasses.root}`]: { alignSelf: "start" },
          [`& .${formControlLabelClasses.labelPlacementStart}`]: {
```